app/controllers/api/subcollections/container_images.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Api
+  module Subcollections
+    module ContainerImages
+      def container_images_scan_resource(provider_name, resource_name = nil, image_id = nil, _payload = nil)
+        raise BadRequestError, "Must specify an id for scanning a #{provider_name} resource" unless resource_name
+        image = ContainerImage.find(image_id)
+        image_name = image.name
+        task = nil
+        begin
+          task = image.scan
+        rescue => bang
+          raise ApiError, "Error during 'Analysis' of #{image_name}: #{bang.message}"
+        end
+        { :msg => "Analysis of #{image_name} successfully initiated", :task_id => task}
+      end
+
+      private
+
+      def container_image_ident(vm)
```

Comment:
Is this method being used? I'm not sure I see it being used. Same for the other private methods